shared/login/signup/success/index.render.desktop.js

```diff
@@ -22,61 +22,61 @@ export default class Render extends Component {
 
   render () {
     return (
-      <Container onBack={this.props.onBack} style={styles.container}>
-        <Text type='Header' style={styles.header}>{this.props.title || 'Congratulations, you’ve just joined Keybase!'}</Text>
-        <Text type='Body' style={styles.body}>Here is your unique paper key, it will allow you to perform important Keybase tasks in the future. This is the only time you’ll see this so be sure to write it down.</Text>
-        <div style={styles.paperKeyContainer}>
-          <Text type='Body' style={styles.paperkey}>{this.props.paperkey.stringValue()}</Text>
-          <Icon type='paper-key-corner' style={styles.paperCorner} />
-        </div>
-        <Checkbox style={styles.check} label='Yes, I wrote this down.' checked={this.state.inWallet} onCheck={inWallet => this.setState({inWallet})} />
-        <Button style={styles.button} type='Primary' label='Done' onClick={() => this.props.onFinish()} disabled={!this.state.inWallet} />
+      <Container onBack={this.props.onBack} style={stylesContainer}>
+        <Text type='Header' style={stylesHeader}>{this.props.title || 'Congratulations, you’ve just joined Keybase!'}</Text>
+        <Text type='Body' style={stylesBody}>Here is your unique paper key, it will allow you to perform important Keybase tasks in the future. This is the only time you’ll see this so be sure to write it down.</Text>
+        <Box style={stylesPaperKeyContainer}>
+          <Text type='Body' style={stylesPaperkey}>{this.props.paperkey.stringValue()}</Text>
+          <Icon type='paper-key-corner' style={stylesPaperCorner} />
+        </Box>
+        {this.props.onFinish && <Checkbox style={stylesCheck} label='Yes, I wrote this down.' checked={this.state.inWallet} onCheck={inWallet => this.setState({inWallet})} />}
```

Comment:
if you don't pass an onFinish it assumes you're just showing a paperkey and not forcing them to do the whole checkbox/ last-part-of-provisioning thing